lib/types/luks.nix

```diff
@@ -3,6 +3,8 @@ let
   keyFile =
     if lib.hasAttr "keyFile" config.settings
     then config.settings.keyFile
+    else if config.askPassword
+    then ''<(echo -n "$password")''
```

Comment:
This will print the password again: https://github.com/nix-community/disko/issues/385